tests/report/display/test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+        rlRun "run_dir=$tmp/original"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check guest display modes"
+        # Two cases for multihost, one with a single guest, other with multiple, to test the default "auto"...
+        rlRun -s "tmt run -av --scratch --id $tmp test -n pass report -h display"
+        rlAssertGrep "^            pass /test/pass$" "$rlRun_LOG"
```

Comment:
```suggestion
        rlAssertGrep "pass /test/pass$" "$rlRun_LOG"
```

Perhaps don't insist on exact number of spaces? So that we don't have to adjust the test if we shift things a bit to the left or right in the future?

Reply:
Fixed in 8d37bce3.